wadf.conf

```diff
@@ -124,6 +124,9 @@ db_name = @underscore:instance@@db_number@
 db_user = @underscore:instance@
 db_pass = @underscore:instance@pass
 db_type = mysql
+; the following setting allows connections to remote mysql8+ servers
+; this value will be used for the host for the db user we create
+db_user_host = localhost
```

Comment:
The default should be '%' (which we can use, as MySQL does, to mean "any"), to be consistent with earlier version.

Reply:
I've updated the default value.